project/Dependencies.scala

```diff
@@ -27,7 +27,7 @@ object Dependencies {
     // Scala
     val circe            = "0.11.0"
     val catsEffect       = "1.1.0"
-    val schemaddl        = "0.9.0"
+    val schemaddl        = "0.10.0-M3"
```

Comment:
Just in case - I recommend you to have separate "Bump X (close #Y)" commit, where you upgrade to a final version and "bumps" with milestones. It will be easier to rebase.